### DISCUSSION TITLE
404 error in all GitHub

### DISCUSSION THREAD

**Discussion by Neustradamus** [community/community#73343] [Repositories]
### Select Topic Area

Bug

### Body

Dear GitHub team,

I have a lot of 404 error pages in GitHub, there is a problem?

Thanks in advance.

**Comment by jpellman** [comment]
There was an outage: https://www.githubstatus.com/incidents/xb30mby9fs5x

**Reply by Neustradamus** [reply]
@jpellman: Thanks for your comment :)
